glean_parser/swift.py

```diff
@@ -157,6 +184,9 @@ def output_swift(
         - namespace: The namespace to generate metrics in
         - glean_namespace: The namespace to import Glean from
         - allow_reserved: When True, this is a Glean-internal build
+        - with_buildinfo: If "true" the `GleanBuildInfo` is generated.
+          Otherwise generation of that file is skipped.
+          Defaults to "true".
```

Comment:
Missing the actual `build_info` param documentation.

Reply:
Kotlin is missing both params documentation.

Reply:
Oh damn, will fill it in.